ang/crmExt.js

```diff
@@ -2,5 +2,9 @@
   // Declare a list of dependencies.
   angular.module('crmExt', [
     'crmUi', 'crmUtil', 'ngRoute', 'dialogService'
-  ]);
+  ])
+    // Make utilities globally available in the app
+    .run(function($rootScope) {
+      $rootScope.ts = CRM.ts('org.civicrm.extensionsui');
```

Comment:
@twomice: Note that I've used the long name of the extension as the argument to `ts()` because civix [does so](https://github.com/twomice/org.civicrm.extensionsui/blob/a35928434db2c297c24a1c9bf8477b2e6ab333b0/extensionsui.civix.php#L29) in its `CRM_*_ExtensionUtil` class. I'm not sure how we started using the short one on the client side, and I'm not sure that it's _wrong_, but there's something be said for consistency, no?